### PR TITLE
chore(Tooltip): change children check

### LINKED ISSUE #2451: Tooltip to strict on checking children
### Description of the bug

**Reported by team in Mattilsynet;**

Vi har denne Tooltip varianten
```
        <Tooltip content="Last ned elektronisk signert PDF" portal={false} placement="bottom">
          <Group>
            <CloudDownFillIcon aria-hidden fontSize="1.5rem" fill={mattilsynetTheme.primary} />
            {label}
          </Group>
        </Tooltip>
```
Dette treffer det du la til på Tooltip
```
    if (
      !children ||
      children?.type === Fragment ||
      (children as unknown) === Fragment
    ) {
      console.error(
        '<Tooltip> children needs to be a single ReactElement and not: <Fragment/> | <></>',
      );
      return null;
    }
```
Dersom vi bytter `<Group>` med `<div>` går det fint, eller wrapper `<Group>` med `<div>`, men det som er rart er at `<Group>` selv returnerer en `<div>` :sweat_smile:

**Suggestion:**
Less strict `children` checking in component, but rather warning added in a `useEffect` where we can inspect the actual amount of rendered DOM children.

### LINKED ISSUE #2724: Allow Tooltip to have string as child element

### PR DESCRIPTION
**Pull request by Barsnes**
resolves #2451
resolves #2724

In a later PR we will experiment with using the popover api.

### CODE REVIEW

File: packages/react/src/components/Tooltip/Tooltip.mdx [hunk 1]
```diff
@@ -7,9 +7,30 @@ import * as TooltipStories from './Tooltip.stories.tsx';
...
 
+`Tooltip` brukes for å vise korte forklaringer til brukeren. 
+
 <Primary />
 <Controls />
 
+## Slik bruker du `Tooltip`
+
+Du kan enten sende inn en string eller et element som `children`. Dersom du sender inn et element, vil det elementet rendres i tooltipen.
+Dersom du bruker kun en string, vil `Tooltip` rendre en `span`.
```
Comment (Barsnes) [inline packages/react/src/components/Tooltip/Tooltip.mdx:18]: Some formulation help here is appreciated

Comment (eirikbacker) [reply packages/react/src/components/Tooltip/Tooltip.mdx]: How about this? Maybe this is enough :D 
```
Du kan enten sende inn en `string` eller et element som `children`. Dersom du sender inn en `string`, vil denne legges i en `<span>`.
```

Comment (Barsnes) [reply packages/react/src/components/Tooltip/Tooltip.mdx]: Yes looks and sound good, traight to the point

File: packages/react/src/components/Tooltip/Tooltip.tsx [hunk 2]
```diff
@@ -14,23 +14,23 @@ import {
...
+import { Fragment, forwardRef, useState } from 'react';
 import type { PortalProps } from '../../types';
 
 export type TooltipProps = {
   /**
    * The element that triggers the tooltip.
    * @note Needs to be a single ReactElement and not: <Fragment/> | <></>
    */
-  children: ReactElement & RefAttributes<HTMLElement>;
+  children: (ReactElement & RefAttributes<HTMLElement>) | string;
```
Comment (mimarz) [inline packages/react/src/components/Tooltip/Tooltip.tsx:33]: Suggest updating the description to match that it supports a string now.

**Review by mimarz — COMMENTED** [review]
Maybe a new story with string as children? It will be good to have for our visual testing also :)

**Review by mimarz — APPROVED** [review]
LGTM 👍

**Pull request merged**